relation_engine_server/README.md

```diff
@@ -460,9 +460,9 @@ make test
 
 ## Deployment
 
-The docker image is pushed to Docker Hub when new commits are made to master. The script that runs when pushing to docker hub is found in `hooks/build`.
+Deploy the image by running the `scripts/docker_deploy` script.
 
-Alternatively, set the image name in `scripts/local-build.sh` and run it to build and deploy locally, which may be a lot faster.
```

Comment:
This comment is still valid -- why are you deleting it?

Reply:
It's no longer called `local-build.sh`, and I don't know whether or not it is faster compared to Github Actions. It was definitely faster compared to the Docker Hub build hook, but we no longer use that.

Reply:
Also, we no longer set the image name within the script